test/ZoraNFTCreatorV1.t.sol

```diff
@@ -85,6 +87,59 @@ contract ZoraNFTCreatorV1Test is Test {
         assertEq(drop.totalSupply(), 10);
     }
 
+    function test_fork_CreateEdition() external {
```

Comment:
should we put the fork test in another file?

Reply:
done